src/Sulu/Component/Content/Document/Subscriber/ShadowCopyPropertiesSubscriber.php

```diff
@@ -216,7 +216,7 @@ private function getLocale(string $propertyName): string
     {
         \preg_match('/i18n:(?P<locale>.+)-shadow-base/', $propertyName, $match);
 
-        return $match['locale'];
+        return $match['locale'] ?? '';
```

Comment:
Here a better error handling would be nice. Or just asserting that this property exists.